src/main/java/org/ohdsi/circe/cohortdefinition/MeasurementOperand.java

```diff
@@ -0,0 +1,16 @@
+package org.ohdsi.circe.cohortdefinition;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class MeasurementOperand {
+  @JsonProperty("Measurement")
+  public Measurement measurement;
+  @JsonProperty("Operator")
+  public String operator;
+  @JsonProperty("Limit")
+  public String limit = "First";
+  @JsonProperty("ValueAsNumber")
+  public NumericRange valueAsNumber;
+  @JsonProperty("SameVisit")
+  public Boolean sameVisit = true;
```

Comment:
I'm not sure this needs to be true by default.   Most of the time, we default boolean values to false.